lib/rouster.rb

```diff
@@ -212,7 +212,15 @@ def initialize(opts = nil)
           @logger.debug(':instance specified, will connect to existing OpenStack instance')
           inst_details = self.ostack_describe_instance(@passthrough[:instance])
           raise ArgumentError.new(sprintf('No such instance found in OpenStack - %s', @passthrough[:instance])) if inst_details.nil?
-          @passthrough[:host] = inst_details.addresses["NextGen"][0]["addr"]
+          if defined?(inst_details.addresses["NextGen"][0]["addr"])
```

Comment:
* can you switch to a more ruby-ish use of `.first` instead of `[0]`?
  * can you use single quotes instead of double?